cl/search/fixtures/florida_state_court_data.json

```diff
@@ -0,0 +1,3726 @@
+[
+  {
+    "fields": {
+      "jurisdiction": "ST",
+      "position": 355.05,
+      "short_name": "",
```

Comment:
I don't know if `short_name` is used much. Regardless, I suppose we can just put a copy/paste of the full_name field into it if we don't want to go to the trouble of figuring it out for each court. Want to poke around and see how much the field is used?

Reply:
Oh, I'm seeing at second glance that it's populated elsewhere. Let's make sure it's consistent.